packages/bundle/src/adaptiveCards/Attachment/ReceiptCardAttachment.js

```diff
@@ -146,10 +143,7 @@ ReceiptCardAttachment.propTypes = {
       vat: PropTypes.string
     }).isRequired
   }).isRequired,
-  language: PropTypes.string.isRequired,
-  styleSet: PropTypes.shape({
-    options: PropTypes.any.isRequired
-  }).isRequired
+  language: PropTypes.string.isRequired
```

Comment:
Why wouldn't you use the `useLanguage` hook?

Reply:
Because it is not in this PR (part 2) yet, it's in part 3.